hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/components/HawkbitUIErrorHandler.java

```diff
@@ -38,10 +42,21 @@ public void error(final ErrorEvent event) {
 
         final Optional<Page> originError = getPageOriginError(event);
 
+        final HawkbitErrorNotificationMessage message = buildNotification(getRootExceptionFrom(event));
```

Comment:
The '''buildNotification''' method logs the error and creates the notification message. If we don't want this behavior the method can be overwritten (e. g. creating a notification message but not logging the error). Default is logging the error. If I remove the log somewhere else, we cannot overwrite it anymore in the current implementation.